### PR TITLE
Add Redux and Store 

### PR DESCRIPTION
**Pull request by hsztan**
I create a Redux store by creating two files 📂, one for books 📚 and another for categories. In those files, I define the action creators, actions, and reducers ⚙️ for each store slice. Then I combine the reducers with the `combineReducers` method. 😎

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const ADD_BOOK = 'book/ADD_BOOK';
+const DELETE_BOOK = 'book/DELETE_BOOK';
+
```
Comment (Gambit142) [inline src/redux/books/books.js]: - [x] Kindly create actions using the _**duck pattern**_. For example, `const ADD_BOOK = 'bookstore/book/ADD_BOOK'`. You can read this [article](https://github.com/erikras/ducks-modular-redux) for more information. 👍

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'category/CHECK_STATUS';
+
```
Comment (Gambit142) [inline src/redux/categories/categories.js]: - [x] Kindly create actions using the _**duck pattern**_. For example, `const CHECK_STATUS = 'bookstore/category/CHECK_STATUS'`. You can read this [article](https://github.com/erikras/ducks-modular-redux) for more information. 👍

File: src/redux/configureStore.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
```
Comment (Gambit142) [inline src/redux/configureStore.js]: - [x] Kindly import `combineReducers` and `createStore` from `'redux'`. Currently, you are importing `configureStore` from `'@reduxjs/toolkit'`. Remember that the essence of this milestone is to use `react-redux` and not `redux/toolkit`. Kindly implement the change. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @hsztan 👋, 

Good job so far!


## Highlights
- Installed `react-redux` ✔️ 
- Used the _**duck pattern**_ ✔️ 
- Created **_action creators_** the correct way ✔️ 
- Set up the **_reducers_** the correct way ✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/redux/categories/categories.js (hunk 2 above)
Comment (hsztan) [reply src/redux/categories/categories.js]: Will do 👍

File: src/redux/configureStore.js (hunk 3 above)
Comment (hsztan) [reply src/redux/configureStore.js]: @Gambit142 Thank you very much for your feedback. I will change this now.

**Review by codepantha — APPROVED** [review]
Hi @hsztan 👋,

## STATUS: APPROVED ✔️  

You have done well implementing all the required changes 👏 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**